projects/timer_challenge/src/main.c

```diff
@@ -0,0 +1,38 @@
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+void counter_increment(SoftTimerId timer_id, void *context) {
```

Comment:
This is a private function, so it should be marked `static` (which makes it not visible from other files) and prefixed with `prv_`.